src/app/shared/dovetail/dovetail-audio.ts

```diff
@@ -96,6 +108,10 @@ export class DovetailAudio extends ExtendableAudio {
     }
   }
 
+  get paused(): boolean {
+    return this._audio.paused && !this._dovetailLoading;
```

Comment:
Underlying `HTMLAudioElement.paused` was causing quite a bit of flicker.  When DT returns the 406, the audio would become paused, and then go back to playing after we make the 2nd request to DT-debug.

As it is, I do see a bit of flicker if the first arranged file is a 0-length mp3.  But I think that was an existing issue, so I didn't try to fix it here.